setup.py

```diff
@@ -21,6 +21,11 @@
     # This is needed due to https://issues.apache.org/jira/browse/BEAM-3357.
     'grpcio>=1.3.5,<=1.7.3',
     'apache-beam[gcp]>=2.2',
+    # Note that adding 'google-api-python-client>=1.6' causes some dependency
+    # mismatch issues if using 'setup.py install'. See Issue #71.
```

Comment:
nit: dependency issues are there no matter we use `setup.py install` or not, the difference seems to be that `setup.py install` stops while `pip install` continues but still installs conflicting versions.

Reply:
Done.